packages/react-monaco-editor/webpack.config.js

```diff
@@ -1,32 +0,0 @@
-// @ts-check
-const path = require('path');
```

Comment:
This file was only used to create a UMD bundle of the editor, which as explained in the PR description, is probably not very useful.